frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/base/built_in_page_widget.dart

```diff
@@ -155,11 +155,9 @@ class _BuiltInPageWidgetState extends State<BuiltInPageWidget> {
           onSelected: (action, controller) async {
             switch (action.inner) {
               case _ActionType.viewDatabase:
-                getIt<TabsBloc>().add(
-                  TabsEvent.openPlugin(
-                    plugin: viewPB.plugin(),
-                    view: viewPB,
-                  ),
+                getIt<PanesCubit>().openTab(
+                  plugin: viewPB.plugin(),
+                  view: viewPB,
```

Comment:
I guess we could simplify this and just pass the `viewPB`, are there any cases where we only have the plugin?

Since you're touching this we can refactor if need be.

Reply:
I likely moved all of existing tabs logic without looking too much into it, I think we should refactor it, I suppose for opening trash we are using plugin.

Reply:
We only need the trash view/plugin once globally I believe? Maybe it's worth looking into making it persist and be available across an app lifecycle..

I know it's not really part of your task, but if you do this let me know, otherwise I'll put it on my list of things to check if we should do.

Reply:
Hmm, I will likely do something about it since I will refactor parts of it to implement drag and drop for tabs and panes.